crown-court-proceeding/src/main/resources/schemas/response/apiDetermineMagsRepDecisionResponse.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "apiDetermineMagsRepDecisionResponse.json",
+  "type": "object",
+  "title": "Determine Mags Rep Decision Response Schema",
+  "properties": {
+    "decisionResult": {
+      "type": "object",
+      "description": "The result of the decision",
+      "existingJavaType": "uk.gov.justice.laa.crime.crowncourt.model.MagsDecisionResult"
+    },
+    "timestamp": {
```

Comment:
There is a timestamp field inside MagsDecisionResult, do we need this? Also this timestamp is not mapped in the MagsProceedingController.